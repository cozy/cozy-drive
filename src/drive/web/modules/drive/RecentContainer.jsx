import React from 'react'

import Container from './Container'

const RecentContainer = props => {
  return (
    <Container
      isTrashContext={false}
      canSort={false}
      canDrop={false}
      canUpload={false}
      canCreateFolder={false}
      canMove={false}
      withFilePath
      {...props}
    />
  )
}

export default RecentContainer
