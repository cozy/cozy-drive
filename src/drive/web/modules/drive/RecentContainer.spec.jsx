import { shallow } from 'enzyme'
import React from 'react'

import RecentContainer from './RecentContainer'

describe('RecentContainer', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<RecentContainer additionalProp="foobar" />)
    expect(wrapper).toMatchSnapshot()
  })
})
