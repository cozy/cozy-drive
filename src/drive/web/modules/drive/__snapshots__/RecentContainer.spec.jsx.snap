// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecentContainer should match the snapshot 1`] = `
<FolderViewWithSharingContext
  additionalProp="foobar"
  canCreateFolder={false}
  canDrop={false}
  canMove={false}
  canSort={false}
  canUpload={false}
  isTrashContext={false}
  withFilePath={true}
/>
`;
